### PR TITLE
RavenDB-6716 REVIEW ONLY - not ready to merge yet

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Client/Documents/Commands/GetDeleteDocumentsThatHaveRevisionsCommand.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Queries;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Commands
+{
+    public class GetDeleteDocumentsThatHaveRevisionsCommand : RavenCommand<QueryResult>
```
Comment (ayende) [inline src/Raven.Client/Documents/Commands/GetDeleteDocumentsThatHaveRevisionsCommand.cs:15]: `GetZombiedRevisionsCommand`, I think

File: src/Raven.Client/Documents/Commands/GetDeleteDocumentsThatHaveRevisionsCommand.cs [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+        public GetDeleteDocumentsThatHaveRevisionsCommand(DocumentConventions conventions, JsonOperationContext context, string indexName, IndexQuery indexQuery, HashSet<string> includes = null, bool metadataOnly = false, bool indexEntriesOnly = false)
+        {
+            if (indexQuery == null)
+                throw new ArgumentNullException(nameof(indexQuery));
+
+            _conventions = conventions ?? throw new ArgumentNullException(nameof(conventions));
+            _context = context ?? throw new ArgumentNullException(nameof(context));
+            _indexName = indexName ?? throw new ArgumentNullException(nameof(indexName));
+            _indexQuery = indexQuery;
```
Comment (ayende) [inline src/Raven.Client/Documents/Commands/GetDeleteDocumentsThatHaveRevisionsCommand.cs:33]: Why the query/ includes / etc ?

File: src/Raven.Client/Documents/Commands/GetRevisionsCommand.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            var request = new HttpRequestMessage
+            {
+                Method = HttpMethod.Get,
+            };
+
+            var pathBuilder = new StringBuilder(node.Url);
+            pathBuilder.Append("/databases/")
+                .Append(node.Database)
+                .Append('/')
+                .Append(_deleteDocumentsWithRevision ? "delete-documents-with-revisions" : "revisions")
```
Comment (ayende) [inline src/Raven.Client/Documents/Commands/GetRevisionsCommand.cs:37]: `revisions/zombied`

File: src/Raven.Client/Documents/Commands/GetDeleteDocumentsThatHaveRevisionsCommand.cs (hunk 1 above)
Comment (fitzchak) [reply src/Raven.Client/Documents/Commands/GetDeleteDocumentsThatHaveRevisionsCommand.cs:15]: This class was removed

File: src/Raven.Client/Documents/Commands/GetDeleteDocumentsThatHaveRevisionsCommand.cs (hunk 2 above)
Comment (fitzchak) [reply src/Raven.Client/Documents/Commands/GetDeleteDocumentsThatHaveRevisionsCommand.cs:33]: This class should have been deleted and not make it to the commit

**Pull request closed without merging**